### DISCUSSION TITLE
Export Folder Datetime request

### DISCUSSION THREAD

**Discussion by Overlordkiwi** [Micke-K/IntuneManagement#269] [Ideas]
2 export options Request
1. Append date time to company name c:\im\company-%datetime%
2. Export to subfolder named for the datetime  c:\company\%datetime%
3.

**Comment by Micke-K** [comment]
Hello,

The following variables are already supported in the Folder name:

* Date
* DateTime
* Organization

You can update the bulk export json file and add a variable to the path eg:

![image](https://github.com/user-attachments/assets/3f22e331-c7b7-4bed-a8c4-d54b91d6ee6d)

or

![image](https://github.com/user-attachments/assets/e633c49c-f2be-4bc6-baa6-ccfe831ad1d0)

Cheers,

**Reply by Overlordkiwi** [reply]
That's great does it work in the UI also, was it in the documentation? Should be a tool tip atleast.

**Comment by bcrabbadmin** [comment]
is it possible to have an export variable to include the baseline name in the file name (ie settings catalog, terms of use, conditional access, etc)? When exporting for policy references, we often do this ad-hoc on a per policy basis, and have had several instances we're we've overwritten previous documents because of the same file name
